packages/gasket-plugin-middleware/lib/utils.js

```diff
@@ -52,30 +44,31 @@ function applyMiddlewareConfig(middleware, plugin, middlewareConfig, middlewareP
   const configEntry = middlewareConfig.find(mw => mw.plugin === pluginName);
 
   if (configEntry) {
-    middleware.paths = configEntry.paths || [];
+    // @ts-ignore
```

Comment:
Why ignore these? If you want to build objects of this type, shouldn't you define that type?